services/records/record_templates/schemas.go

```diff
@@ -0,0 +1,161 @@
+package record_templates
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var validRecordTemplatesConstraintTypes = []string{"UNKNOWN", "PASSED", "FAILED"}
+
+var recordTemplateSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+		ForceNew: true,
+	},
+	"description": {
+		Type:     schema.TypeString,
+		Optional: true,
+		ForceNew: true,
+	},
+	"record_state": {
+		Type:         schema.TypeString,
+		Optional:     true,
```

Comment:
Shouldn't it be computed? It seems to be absent from the api request [here](https://leanspace-dev.readme.io/reference/createrecordtemplate)